Riva0110/Vocab-Garden
# Feature/friends

### Description

* Add  friend's system
* Add invitation's system for battle review
* Monitor friends online status

### Test Plan

* Check if user can send request, and accept or reject a friend request
* Check if the user status show the lasted information
